Application/Sources/Helpers/Environment.swift

```diff
@@ -5,6 +5,7 @@
 
 import Foundation
 
+@frozen
```

Comment:
Why is this needed? Is it just to avoid the linter warnings? If so, I usually just add `disable/enable` comments in the enum's body.